### PR TITLE
Fix: broken doc links

### PR DESCRIPTION
**Pull request by kanakgautam**
Signed-off-by: kanak gautam <kanakgautam5@gmail.com>

Fixed broken links in [Using OCIO](https://opencolorio.readthedocs.io/en/latest/guides/using_ocio/using_ocio.html#c):
1. http://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/export/OpenColorIO -> http://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/src/OpenColorIO
2. [Developer Guide](https://opencolorio.org/developers/index.html) -> [Developer Guide](https://opencolorio.readthedocs.io/en/latest/guides/developing/developing.html#developing-with-ocio)
3. [Usage Examples](https://opencolorio.org/developers/usage_examples.html) -> [Usage Examples](https://opencolorio.readthedocs.io/en/latest/guides/developing/developing.html#usage-examples)

### CODE REVIEW

File: docs/guides/using_ocio/compatible_software.rst [hunk 1]
```diff
@@ -62,7 +62,7 @@ C++
 
 The core OpenColorIO API is available for use in C++. Minimal code examples are also available in the source code distribution. 
 
-Website: `<http://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/export/OpenColorIO>`__
+Website: `<http://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/src/OpenColorIO>`__
```
Comment (michdolan) [inline docs/guides/using_ocio/compatible_software.rst:65]:
```suggestion
Website: `<http://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/include/OpenColorIO>`__
```
`export` was replaced with `include` in the new repo structure.

**Review by michdolan — CHANGES_REQUESTED** [review]
Thanks for the contribution @kanakgautam

File: docs/guides/using_ocio/compatible_software.rst (hunk 1 above)
Comment (kanakgautam) [reply docs/guides/using_ocio/compatible_software.rst]: Thanks! I've made the fix.

**Review by hodoulp — APPROVED** [review]

**Comment by hodoulp** [comment]
@michdolan, @kanakgautam did the requested change, could you now approve the pull request?

**Comment by hodoulp** [comment]
Friendly reminder that the pull request can now be merged next week.

**Review by doug-walker — APPROVED** [review]

**Comment by hodoulp** [comment]
@michdolan The pull request seems fine now, could you approve it ?

**Comment by hodoulp** [comment]
Hi @michdolan, could you review/approve the pull request?

**Review by michdolan — APPROVED** [review]

**Pull request merged**